src/components/HTMLEngineProvider/htmlEngineUtils.js

```diff
@@ -62,7 +62,7 @@ function isChildOfComment(tnode) {
  * @returns {Boolean}
  */
 function isChildOfH1(tnode) {
-    return isChildOfNode(tnode, (node) => node.domNode.name.toLowerCase() === 'h1');
+    return isChildOfNode(tnode, (node) => node.domNode.name === 'H1');
```

Comment:
After checking the tag name is not in uppercase, not sure why though. Was there a reason you had to use toLowerCase?

Reply:
I've already checked that before it was lowercase somehow. That's why I've just used 'h1' and @roryabraham suggested to use toUpperCase() here: https://github.com/Expensify/App/pull/30876#issuecomment-1800955345